backend/cmd/server.go

```diff
@@ -13,16 +13,23 @@ func main() {
 	// @todo: Make this a uint and validate the values
 	port := flag.String("port", "4466", "Port to listen from")
 	pluginDir := flag.String("plugins-dir", "./plugins", "Specify the plugins directory to build the backend with")
+	// For inCluster config we need to get the oidc properties from the flags
+	oidcClientID := flag.String("oidc-client-id", "", "ClientID for OIDC")
+	oidcClientSecret := flag.String("oidc-client-secret", "", "ClientSecret for OIDC")
+	oidcIdpIssuerURL := flag.String("oidc-idp-issuer-url", "", "Identity provider issuer URL for oidc")
```

Comment:
You should validate the use of this with `useInCluster`. i.e. if `useInCluster` is false but we are using the `oidc` options, there should be an error.